### ISSUE TITLE
Bug in evaluation?

### ISSUE THREAD

**Issue by gitzyong812** [coldmanck/VidHOI#15]
![image](https://user-images.githubusercontent.com/11305113/224236371-50f2216e-c04a-49be-b901-91a1fef951d1.png)
in file https://github.com/coldmanck/VidHOI/blob/master/vidor_eval.ipynb. 

We should not just skip these triplets. Instead, ap for these triplets should be set as 0.

**Comment by coldmanck** [comment]
Good catch, that makes sense. Please feel free to add a PR and report the revised numbers.
